src/ucp/wireup/select.c

```diff
@@ -1258,14 +1260,15 @@ ucp_wireup_iface_avail_bandwidth(const ucp_worker_iface_t *wiface,
     /* Apply dev num paths ratio after fp8 pack/unpack to make sure it is not
      * neglected because of fp8 inaccuracy
      */
-    local_bw *= ucp_tl_iface_bandwidth_ratio(
-                   context, local_dev_count[dev_index],
-                   wiface->attr.dev_num_paths);
+    count     = dev_count->local[dev_index] - dev_count->local_skip[dev_index],
```

Comment:
in theory `dev_count->local` can be smaller than dev_count->local_skip
maybe add an assert?

Reply:
added for local and remote

Reply:
rename count to path_index, also in ucp_tl_iface_bandwidth_ratio function